src/pages/settings/Security/SecuritySettingsPage.js

```diff
@@ -58,6 +61,14 @@ const SecuritySettingsPage = (props) => {
                             shouldShowRightIcon
                         />
                     ))}
+                    <MenuItem
```

Comment:
What's this for? It looks like a button you were maybe using for local testing ... any reason we should deploy it to staging?

Reply:
oh me oh my i'll remove that